lua/cfc_notifications/shared/sh_base.lua

```diff
@@ -8,6 +8,10 @@ CFCNotifications.PRIORITY_NORMAL = 3
 CFCNotifications.PRIORITY_HIGH = 4
 CFCNotifications.PRIORITY_MAX = 5
 
+CFCNotifications.ALIGN_LEFT = 1
+CFCNotifications.ALIGN_CENTER = 2
+CFCNotifications.ALIGN_RIGHT = 3
```

Comment:
Should we not be using https://wiki.facepunch.com/gmod/Enums/TEXT_ALIGN
I get that they have additional alignment stuff, but thats where we throw an error.
Feel like having to put `CFCNotifications` before the align thing makes the line a little long
Not entirely convinced tho so idm discussing this

Reply:
I see what you mean about decreasing the length, but I think it's better if we make it directly clear what alignments are possible, without having to go out of the way to say "hey you specifically can't use these two alignment styles," and require people to directly read that on the docs or in an error to know what is and isn't allowed.

Reply:
Yeah I can see that, aiight leave as is